### ISSUE TITLE
Add scientific units to cell mass

### ISSUE THREAD

**Issue by hhyyrylainen** [Revolutionary-Games/Thrive#1707]
It should be pretty easy to look up masses of the different organelles and then add the unit to the GUI for displaying the cell mass.

**Comment by FuJa0815** [comment]
Do we know how much volume one hex contains?

**Comment by hhyyrylainen** [comment]
The hexes are volume, it's not necessarily correlated so that bigger organelles are heavier. Looking up on wikipedia how much mitochondria weight etc. I think is the right way to go.

**Comment by FuJa0815** [comment]
Yes, I can look up the density, but to calculate the weight, I need to know how much volume one hex contains

**Comment by hhyyrylainen** [comment]
Assume that the organelle is the average size said on wikipedia?

**Comment by FuJa0815** [comment]
Ok, just wanted to make sure that the volume was not defined before.

**Comment by Holomanga** [comment]
http://book.bionumbers.org/how-big-is-an-e-coli-cell-and-what-is-its-mass/

So 1 µm^3 ~ 1 pg because cells are water, and conveniently E. coli are 1 µm^3. Mitochondria are about the same size (around 2 µm in length (http://book.bionumbers.org/how-big-are-mitochondria/)), and are 0.3 mass units in Thrive. So 1 Thrive mass unit ~ 3.3 picograms.

Is this mass to be displayed in the Organism Statistics panel in the top right of the editor?

**Comment by hhyyrylainen** [comment]
I'm closing this as this was not on the Microbe roadmap and we have completed that stage of the game development.
